packages/react-static/src/static/getRoutes.js

```diff
@@ -46,12 +46,9 @@ export default async function getRoutes(state, callback = d => d) {
     if (!has404 && !incremental) {
       allNormalizedRoutes.unshift({
         path: '404',
-        template: path.relative(
-          state.config.paths.ROOT,
```

Comment:
Why was this removed?

`resolve` should probably _still_ use this `root` path?

(p.s. should now be running travis again).

Reply:
The result of `path.resolve(__dirname, path.join('..', 'browser', 'components', 'Default404'))` is an absolute path, which always works fine when passed to `template`. 

Adding `path.relative(ROOT, <path>)` will calculate `<path>` relative to `ROOT`, so the path will be the same, just that now it will have a ton of `../../../` to point to the same directory from the point of view of `ROOT`. 

In the end, the result should be the same, as long as your current working directory is the same as `ROOT`. After this change, I believe everything that worked in the past should still work (lets see what Travis thinks), and it will now still work even if `process.cwd() !== ROOT`:

```sh
$ node
> path.relative('/foo/bar/baz', '/foo/bar/baz/qux')
'qux'
> path.relative('/foo/bar/baz', '/test/bar/baz/qux')
'../../../test/bar/baz/qux'
```

Reply:
Thanks for the explanation! 

I didn't really mean, why was `path.relative` removed, but why was `state.config.paths.ROOT` removed. I think we used to be able to change the `ROOT` dir. Overall, I don't really care. All the files _should_ live under `state.config.paths.ROOT` anyway, so `.resolve` _should_ always work 🗡

Reply:
@SleeplessByte 

> I think we used to be able to change the ROOT dir

We still can do so. This part of the code only kicks in when the user didn't define a 404 page, and therefore we use the default one that comes with the module. The ROOT dir shouldn't make a difference here.